src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,182 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+/**
+ * Creates a Data Pipeline for backing up Dynamo DB tables within a region.
+ */
+public class DynamoDataPipelineHelper {
+
+    private static final String READ_THROUGHPUT_PERCENTAGE = "0.3";
+
+    /**
+     * Creates data pipeline objects for a daily data pipeline that will export data for DynamoDB tables in a region.
+     *
+     * @param tables       dynamoDB tables to dump
+     * @param dynamoRegion region that the dynamoDB tables belong to
+     * @param s3Bucket     s3 bucket for dynamo table dump and logs, table dumps will go in
+     *                     s3://s3Bucket/regionName/tableName/, logs will go in s3://s3Bucket/logs
+     * @param localTime    time to begin runnin pipeline
+     */
+    public static List<PipelineObject> createPipelineObjects(Region dynamoRegion,
+                                                             Collection<TableDescription> tables,
+                                                             String s3Bucket,
+                                                             LocalTime localTime) {
+        List<PipelineObject> pipelineObjects = Lists.newArrayList();
```

Comment:
Is there any reason to use Lists.newArrayList() over the ArrayList constructor? With the diamond operator, the ArrayList constructor is arguably less wordy.